src/main/java/Commands/CommandInvoker.java

```diff
@@ -0,0 +1,61 @@
+package Commands;/*
+*  "Duke.UI"
+* */
+
+import Exceptions.DukeException;
+
+/**
+ * The type Commands.Command invoker.
+ */
+public class CommandInvoker {
+
+    private static boolean isProgramEnd = false;
+
+    /**
+     * Instantiates a new Commands.Command invoker.
+     */
+    public CommandInvoker(){};
+
+    /**
+     * Invoke userCommand
+     *
+     * @param c the userCommand
+     */
+    CommandInvoker(UserCommand c){
+        Invoke(c);
+    }
+
+
+    /**
+     * Invoke the Usercommand.
+     *
+     * @param c the Commands.UserCommand
+     */
+    public static void Invoke(UserCommand c){
```

Comment:
Consider using lower case, i.e. `invoke` instead of `Invoke` to follow coding standards.

Reply:
yes boss.. thanks for the highlight :)